perception/lidar_centerpoint/config/detection_class_adapter.param.yaml

```
@@ -0,0 +1,38 @@
+/**:
+  ros__parameters:
+    allow_remapping_by_area_matrix:
+      # NOTE(kl): We turn all vehicles into trailers if they go over 3x12 m2.
+      # NOTE(kl): We turn cars into trucks if they have an area between 2.2 x 5.5 and 2.5 * 7.9 m2
+      # row: original class. column: class to remap to
+      #UNKNOWN, CAR, TRUCK, BUS,  TRAILER, MOTORBIKE, BICYCLE,PEDESTRIAN
+      [0,       0,   0,     0,    0,       0,         0,      0,         #UNKNOWN
+       1,       0,   1,     1,    1,       0,         0,      0,         #CAR
+       1,       0,   0,     0,    1,       0,         0,      0,         #TRUCK
+       1,       0,   0,     0,    1,       0,         0,      0,         #BUS
+       1,       0,   0,     0,    0,       0,         0,      0,         #TRAILER
+       1,       0,   0,     0,    0,       0,         0,      0,         #MOTORBIKE
+       1,       0,   0,     0,    0,       0,         0,      0,         #BICYCLE
+       1,       0,   0,     0,    0,       0,         0,      0]         #PEDESTRIAN
```

```suggestion
       0,       0,   1,     1,    1,       0,         0,      0,         #CAR
       0,       0,   0,     0,    1,       0,         0,      0,         #TRUCK
       0,       0,   0,     0,    1,       0,         0,      0,         #BUS
       0,       0,   0,     0,    0,       0,         0,      0,         #TRAILER
       0,       0,   0,     0,    0,       0,         0,      0,         #MOTORBIKE
       0,       0,   0,     0,    0,       0,         0,      0,         #BICYCLE
       0,       0,   0,     0,    0,       0,         0,      0]         #PEDESTRIAN
```
We don't allow for any classes to be mapped to the unknown class.